tests/_template.yml

```diff
@@ -1,47 +1,47 @@
-- name: moo file                     # The name of the workflow (required)
-  command: bash moo_workflow.sh      # The command to execute the workflow (required)
-  files:                             # A list of files to check (optional)
-    - path: "moo.txt"                # File path. (Required for each file)
-      contains:                      # A list of strings that should be in the file (optional)
+- name: moo file # The name of the workflow (required)
```

Comment:
🤣  Love it ❤️ , maybe it can be added to the template in the future.
I might steal it for sarek, we also have a similar set up than you have here for the CI tests and this file would be very helpful for all future CI tests

Reply:
Yes I really like it! there is also a nextflow test framework coming out that could be useful but for now this is very good, especially for a small dev team